src/lib.rs

```diff
@@ -22,11 +22,14 @@ libflac-sys = "0.1"
 # Features
 
 * `ogg` (enabled by default): Support for FLAC data in OGG containers
+* `build` (enabled by default): build instead of linking to libraries from
+   the system
```

Comment:
What happens if the external `libFLAC` is incompatible with the headers used for creating the Rust bindings?

Should we add a warning here?

Reply:
I don't think this is a concern – going by https://packages.debian.org/search?keywords=flac, jessie is on 1.3.0, stretch and buster are on 1.3.2, and sid is on 1.3.3, all of which are compatible. Also https://xiph.org/flac/ has a news sexion, which has these dates on it:
```
4 Aug 2019 : FLAC 1.3.3 released
1 Jan 2017 : FLAC 1.3.2 released
27 Nov 2014: FLAC 1.3.1 released
01 Jun 2013: FLAC 1.3.0 released
```

The likelihood of there being a new incompatible version is too close to zero IMO.

Reply:
OK, thanks, makes sense.